### PR TITLE
support release package to conan 

### PR DESCRIPTION
**Pull request by libbylg**

### CODE REVIEW

**Comment by madebr** [comment]
@libbylg 
Please open the issues page of this repo, so I can report the current test failure.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        os: ["macos-latest", "ubuntu-latest", "windows-latest"]
+        build_shared_libs: ["ON", "OFF"]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: "Checkout repo"
+        uses: actions/checkout@v2
+      - name: "Setup java"
+        uses: actions/setup-java@v1
+        with:
+          java-version: 13
```
Comment (madebr) [inline .github/workflows/ci.yml]:
```suggestion
```

**Comment by libbylg** [comment]
Oh My GOD! It passed!!

**Comment by madebr** [comment]
You seem surprised :stuck_out_tongue: 
Green is the favourite colour of developers :smile:

**Comment by libbylg** [comment]
> You seem surprised 😛
> Green is the favourite colour of developers 😄

Yes, I understand that. It take me several hours. 
What I can not understand is why we have to use the cmake to replace the shell script.

**Comment by madebr** [comment]
You can still use shell script for testing.
Github actions has msys on Windows.

But since you already use cmake for building libproperties, why not use it for testing as well?

**Pull request merged**